src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
+@import './blocks/variables';
+@import './blocks/calendar';
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Try not to use * selector (it affects performance), better use html or body tag.

![image](https://github.com/mate-academy/layout_calendar/assets/54860641/953d768c-7d0f-409a-9c7a-9bf4829df421)